t/lib/dubbo-serialization-backend/dubbo-serialization-backend-interface/pom.xml

```diff
@@ -0,0 +1,46 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Why do we need these files?

Reply:
Due to a bug in Dubbo 2.x that prevents specifying the serialization and deserialization protocol through the client, for the sake of convenient testing, I have fixed Dubbo's serialization method to fastjson. If using the same backend as the Dubbo proxy, correct test results cannot be obtained. I will provide additional information on how to fix the bug in Dubbo serialization and deserialization. At the same time, I need to introduce tests for more parameter types, unlike Dubbo-proxy, which supports only Map types.

Reply:
I'm not familiar with dubbo, so these files act as a backend?

Reply:
yes